testsuite/integration/microprofile/src/test/java/org/wildfly/test/integration/microprofile/health/MicroProfileHealthApplicationStartupTestBase.java

```diff
@@ -42,6 +42,7 @@
 import java.io.IOException;
 import java.net.URL;
 
+
 /**
```

Comment:
these whitespace changes are unnecessary but I'll leave this to the merger to decide if this needs to be fixed